sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/model_batch_deployment.py

```diff
@@ -18,11 +18,13 @@
 from azure.ai.ml.exceptions import ErrorCategory, ErrorTarget, ValidationErrorType, ValidationException
 from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY, PARAMS_OVERRIDE_KEY
 from azure.ai.ml.entities._util import load_from_dict
+from azure.ai.ml._utils._experimental import experimental
 from azure.ai.ml._schema._deployment.batch.model_batch_deployment import ModelBatchDeploymentSchema
 from .model_batch_deployment_settings import ModelBatchDeploymentSettings
 from .code_configuration import CodeConfiguration
 
 
+@experimental
```

Comment:
does `ModelBatchDeploymentSettings` also need to experimental or is it already marked as such?

Reply:
Good catch, will make that change as well